physicsnemo/distributed/manager.py

```diff
@@ -548,30 +548,30 @@ def setup(
                 manager._local_rank = local_rank
 
         manager._device = torch.device(
-            f"cuda:{manager.local_rank}" if torch.cuda.is_available() else "cpu"
+            f"cuda:{manager._local_rank}" if torch.cuda.is_available() else "cpu"
```

Comment:
Any specific reason to switch to using private attributes here instead of the public property versions of these as was used before?

Reply:
If the external ones alias those internal ones, that is OK, but isn't it cleaner to use the internal ones for internal tasks?